app/upgrades/v8_2/constants.go

```diff
@@ -0,0 +1,10 @@
+package v82
+
+const (
+	// UpgradeName is the shared upgrade plan name for mainnet and testnet
```

Comment:
nit, as you pointed out above this is not for testnet

```suggestion
	// UpgradeName is the shared upgrade plan name for mainnet
```